src/reference/modules/administration/pages/server.adoc

```diff
@@ -365,27 +365,30 @@ By default it logs all connections from nodes, to allow proper
 auditing of node connections, but you can disable these logs (or increase the log level).
 To only log errors:
 
-* On systemd systems
-                                                                            
-** Create a `/etc/systemd/system/rudder-cf-serverd.service.d/override.conf` file containing:
+==== On systemd systems
+
+* Create a `/etc/systemd/system/rudder-cf-serverd.service.d/override.conf` file containing:
 
 ----
 [Service]
-Environment=VERBOSITY_OPTION=
+Environment=VERBOSITY_OPTION
 ----
-    
-** Run `systemctl daemon-reload` then `systemctl restart rudder-cf-serverd`
 
-* On systems using the init script
+* Run `systemctl daemon-reload` then `systemctl restart rudder-cf-serverd`
+
+`VERBOSITY_OPTION` possible values: 'error', 'warning', 'info', 'verbose', 'debug'.
```

Comment:
should be `--inform`, `--verbose`, `--debug`

Reply:
or maybe ` --log-level error`, ` --log-level warning`, ` --log-level notice`, ` --log-level verbose`, ` --log-level debug` but these are newer and we should check they are equivalent.